### ISSUE TITLE
Deaktivierung von "Record-Changes only" berücksichtigt weiterhin eingetragene Werte aus "log changes interval(s)" 

### ISSUE THREAD

**Issue by schlaubstar** [ioBroker/ioBroker.influxdb#263]
**Bug:**
Hat man einmal auf Datenpunktebene "Record Changes only" aktiviert und einen Wert bei "log changes interval(s)" befüllt, dann bleibt das Verhalten, dass der letzte bekannte Wert nach Erreichen des eingestellten Intervals 'wiederholt' wird, bestehen, auch wenn man "Record Changes only" deaktiviert.

**To Reproduce**  

1. Auf einem Datenpunkt die Option "Record changes only" aktivieren und bei "log changes interval(s)" einen Wert, bspw. 43.200 eintragen
2. Einstellungen speichern
3. In dem Datenpunkt aus 1. die Option "Record changes only" deaktivieren, dann werden die Eingabefelder für "log changes interval(s) und "Minimum difference from last value" ausgegraut 
4. Einstellungen speichern
5. Dem Datenpunkt nun einen beliebigen Wert zuweisen

**Expected behavior**  
Solange dem Datenpunkt kein neuer Wert (z.B. durch simple-api, javascript, ....) zugewiesen wird, sollte der Datenpunkt keine Änderung erfahren.

Leider wird aber nach dem voreingestellten intervall (im obigen Bepispiel 12h) von der Quelle Influxdb eine Kopie des letzten Datenpunks angelegt.

**Screenshots & Logfiles**  
Konfiguration des Datenpunkts nach Entfernen der Checkbox "Record Changes only"
![2022-05-26 22_46_28-objects - iobroker-pi — Mozilla Firefox](https://user-images.githubusercontent.com/25862995/170578172-434fdc51-3325-450b-9e88-4177be41d987.png)
"Echoing" des letzten Wertes nach 12h ("Log changes interval(s)") durch die Quelle InfluxDB
![2022-05-24 23_10_46-objects - iobroker-pi — Mozilla Firefox](https://user-images.githubusercontent.com/25862995/170578241-11628ed0-59b2-4e64-84b0-1b73ceeb7d4f.png)

**Versions:**  
 - Adapter version: 3.0.2
 - JS-Controller version: 4.0.23
 - Node version: v14.18.3
 - Operating system: Raspbian GNU/Linux 10 (buster)

**Additional context**  
Ich denke die einfachste Lösung wäre es, die beiden Werte für "log changes interval(s)" und "minimum difference from last value" mit dem Wert 0 vorbelegt würden, wenn man die Option "record changes only" deaktiviert - Sauberer wäre es jedoch bei der Berücksichtigung dieser Werte zu prüfen ob überhaupt "record changes only" aktiv ist.

**Comment by Apollon77** [comment]
Good catch .... Genau das fehlte an zwei Stellen ... Bitte mal GitHub version probieren ... (BTW: GitHub Installs können mit npm 6 aktuell 10+ Minuten dauern ... einfach Geduld haben)

**Comment by Apollon77** [comment]
Fixed in 3.1.0
